connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerSinkTask.java

```diff
@@ -210,7 +210,7 @@ public int commitFailures() {
      * Initializes and starts the SinkTask.
      */
     protected void initializeAndStart() {
-        log.debug("Initializing task {} with config {}", id, taskConfig);
+        log.debug("Initializing task {} ", id);
```

Comment:
I don't think it's too important but since the checkstyle issue needs to be resolved we can bump the logging level lower if you wish.